### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by rustycl0ck**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
+## 0.3.0
+
```
Comment (SuperQ) [inline CHANGELOG.md:2]: Let's add a Breaking Changes note here about the JSONPath library since this will require users update their configs.

Comment (rustycl0ck) [reply CHANGELOG.md:2]: Updated :+1:

**Review by SuperQ — APPROVED** [review]
LGTM

**Comment by rustycl0ck** [comment]
Anything else I need to do for releasing this other than merging the PR? I see that the builds completed successfully, but I don't see the `0.3.0` image in the docker registry, and neither do I see any github release by prombot yet.

**Comment by SuperQ** [comment]
Yes, a tag needs to be pushed to github in order to trigger the circleci release process.

**Pull request merged**